jaxrs/src/main/java/org/killbill/billing/jaxrs/resources/SubscriptionResourceHelpers.java

```diff
@@ -93,7 +93,8 @@ public static List<PlanPhasePriceOverride> buildPlanPhasePriceOverrides(final It
         if (priceOverrides != null) {
             for (final PhasePriceJson input : priceOverrides) {
                 Preconditions.checkNotNull(input);
-
+                Preconditions.checkArgument(input.getFixedPrice() != null || input.getRecurringPrice() != null || input.getUsagePrices() != null || (input.getUsagePrices() != null && !input.getUsagePrices().isEmpty()),
```

Comment:
I think we can remove the  `input.getUsagePrices() != null` and simply keep the `(input.getUsagePrices() != null && !input.getUsagePrices().isEmpty()`

Reply:
Addressed in 984a82d